### ISSUE TITLE
[BUG-REPORT] Vaex `to_dask_array` fails `AttributeError: 'ValueError' object has no attribute 'numpy'`

### ISSUE THREAD

**Issue by Ben-Epstein** [vaexio/vaex#1970]
Thank you for reaching out and helping us improve Vaex!

Before you submit a new Issue, please read through the [documentation](https://docs.vaex.io/en/latest/). Also, make sure you search through the Open and Closed Issues - your problem may already be discussed or addressed.

**Description**
Please provide a clear and concise description of the problem. This should contain all the steps needed to reproduce the problem. A minimal code example that exposes the problem is very appreciated.

**Software information**
 - Vaex version (`import vaex; vaex.__version__)`: {'vaex-core': '4.8.0', 'vaex-hdf5': '0.11.1'}
 - Vaex was installed via: pip / conda-forge / from source  pip
 - OS: MacOS

**Additional information**
Please state any supplementary information or provide additional context for the problem (e.g. screenshots, data, etc..).

```
import vaex

def tolist(x,y):
    return list([x,y])

data = {'A':[1,2,1],'B':['a','b','c'], 'C':['d', 'e', 'f']}
df = vaex.from_dict(data)
df.to_dask_array()
```

**Comment by Ben-Epstein** [comment]
Seems like it's any non numeric columns that cause the issue 
```
from random import randint 

df = vaex.from_arrays(
    id=[1,1,4,2,5,64,46,64,4,2,5],
    letter = ["a" for _ in range(11)]
)

df[["id"]].to_dask_array()  # works
df[["id","letter"]].to_dask_array()  # error above
```

**Comment by JovanVeljanoski** [comment]
True, exporting non numeric data to a dask array is not supported (for now).

**Comment by maartenbreddels** [comment]
Yeah, we could improve on the error though. Not sure if we can have a meaningful dask-arrow-array export btw.

**Comment by maartenbreddels** [comment]
I think this is a good first issue to improve the error message.
